### PR TITLE
DEVOPS-671 Adding bors configuration

### PR DESCRIPTION
**Pull request by jorge-cr-13**
Adding the bors configuration making use of the reusable workflow.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,357 +17,23 @@ concurrency:
...
-    needs: [selection, dart-ok, cargo-ok]
-    if: ${{always()}}
+    needs: [dev-ci]
     runs-on: ubuntu-20.04
     timeout-minutes: 5
     steps:
       - name: CI OK Testing
         run: |
-          if (${{ needs.selection.outputs.dart == 'true'}} || ${{ needs.selection.outputs.rust == 'true' }} )
+          if (( (${{needs.dev-ci.outputs.rust_output != 'true'}} || ${{needs.dev-ci.outputs.cargo-ok == 'success'}}) && (${{needs.dev-ci.outputs.dart_output != 'true'}} || ${{needs.dev-ci.outputs.dart-ok == 'success'}}) ) ||
```
Comment (acrrd) [inline .github/workflows/ci.yml:32]: Can't we move these checks once in `ci_reusable_wf` as `ci-ok` and here just refer to that job?

Comment (jorge-cr-13) [reply .github/workflows/ci.yml]: Yes, we can.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -14,357 +17,23 @@ concurrency:
...
-            else
-              exit 1
-            fi
+            exit 0
           else
             exit 1
           fi
-
+          
+
```
Comment (acrrd) [inline .github/workflows/ci.yml:39]:
```suggestion
```

**Review by acrrd — APPROVED** [review]

**Comment by jorge-cr-13** [comment]
bors merge

**Pull request merged**